release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -466,19 +466,115 @@ With this update, the default credentials request for AWS has been modified to a
 
 [id="ocp-4-13-storage-lvms-dual-stack-support"]
 ==== Dual-stack support for {lvms-first}
-
 In {product-title} {product-version}, {lvms} is supported in dual-stack for IPv4 and IPv6 network environments. For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network].
 
 [id="ocp-4-13-storage-lvms-in-zpt"]
 ==== Support for {lvms} in GitOps ZTP
-
 In {product-title} {product-version}, you can add and configure {lvms-first} through GitOps ZTP. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-lvms-installing-using-web-console_ztp-manual-install[Intalling LVM Storage by using the web console], xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-provisioning-lvm-storage_ztp-advanced-policy-config[Configuring LVM Storage using PolicyGenTemplate CRs], and xref:../scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc#lvms-configuring-lvms-on-sno_sno-configure-for-vdu[LVM Storage].
 
 [id="ocp-4-13-storage-lvms-in-disconnected-env"]
 ==== Support for {lvms} in disconnected environments
 
 In {product-title} {product-version}, you can install {lvms} in disconnected environments. For more information, see xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-installing-lvms-disconnected-env_logical-volume-manager-storage[Installing LVM Storage in a disconnected environment].
 
+[id="ocp-4-13-storage-user-managed-encryption"]
+==== User-managed encryption is generally available
+The user-managed encryption feature allows you to provide keys during installation that encrypt {product-title} node root volumes, and enables all managed storage classes to use these keys to encrypt provisioned storage volumes. This allows you to encrypt storage volumes with your selected key, instead of the platform’s default account key.
+
+This features supports the following storage types:
+
+* Amazon Web Services (AWS) Elastic Block storage (EBS) (for more information, see ???)
```

Comment:
Missing link?

Reply:
Yes, the ??? is a placeholder that I use for links when i can't put them in because the content is not merged yet.